provider/core/src/main/java/org/arquillian/pact/provider/core/PactProviderExtension.java

```diff
@@ -11,5 +16,10 @@ public void register(ExtensionBuilder builder) {
                 .observer(PactProviderConfigurator.class)
                 .observer(HttpTargetCreator.class)
                 .service(ResourceProvider.class, HttpTargetResourceProvider.class);
+
+        if(Validate.classExists("org.jboss.arquillian.container.test.spi.client.deployment.AuxiliaryArchiveAppender")) {
```

Comment:
Yes this code comes from Arquillian core itself IIRC I remember it was Aslak who pointed me out. I am not a big fun of this, I would like to see something given by Arquillian itself but for now this is what we have :)